Autopilot/Drivers/Peripherals/Src/altimeter.c

```diff
@@ -0,0 +1,65 @@
+#include "altimeter.h"
+#include "debug.h"
+
+#define MPL3115A2_ADDR 0x60 << 1
+#define MPL3115A2_I2C I2C2
+#define STATUS 0x00
```

Comment:
Add a comment separating these from the other defines. (e.g. // register addresses)